homeassistant/components/sensor/sigfox.py

```diff
@@ -0,0 +1,140 @@
+"""
+Sensor for SigFox devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sigfox/
+"""
+import logging
+import datetime
+import json
+import voluptuous as vol
+import requests
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = datetime.timedelta(seconds=30)
+API_URL = 'https://backend.sigfox.com/api/'
+CONF_API_LOGIN = 'api_login'
+CONF_API_PASSWORD = 'api_password'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_LOGIN): cv.string,
+    vol.Required(CONF_API_PASSWORD): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the sigfox sensor."""
+    api_login = config[CONF_API_LOGIN]
+    api_password = config[CONF_API_PASSWORD]
+    sigfox = SigfoxAPI(api_login, api_password)
```

Comment:
You are logging an error if the login fails. If that's the case then the platform setup should fail too.

Reply:
OK addressed this. Is there not a standard HA way you check credentials of cloud services?